tests/conftest.py

```diff
@@ -215,16 +268,6 @@ def generate_w_logprobs(
             outputs.append((output_ids, output_str, output_logprobs))
         return outputs
 
-    def generate_greedy(
-        self,
-        prompts: List[str],
-        max_tokens: int,
-    ) -> List[Tuple[List[int], str]]:
-        greedy_params = SamplingParams(temperature=0.0, max_tokens=max_tokens)
-        outputs = self.generate(prompts, greedy_params)
-        return [(output_ids[0], output_str[0])
-                for output_ids, output_str in outputs]
-
```

Comment:
why are we segmenting this between VllmRunner and VllmRunnerNm now?

Reply:
moved the functions that I added to `vllm_runner` in the marlin PR to `vllm_runner_nm` --> so now `vllm_runner` perfectly matches the upstream